Assets/PilotToInstructorSynchronization.cs

```diff
@@ -22,6 +23,15 @@ void FixedUpdate()
         {
             return;
         }
+        if (sharedGameState.gameOverSceneTransition.Value)
```

Comment:
It might be the easiest solution to check for this scene transition value every frame, but IMHO it's a little overkill, since the value will be changed exactly once throughout the game. In cases where you have values which are not necessarily part of the "real time" aspect of games, you should use events that fire in the few specific frames where they occur. The advantage of that is efficiency (you perform less idle computation) and more well structured code since `FixedUpdate` or `Update` don't become gigantic methods which are difficult to read.